serverless/src/types.ts

```diff
@@ -0,0 +1,35 @@
+export const SUCCESS = "success";
```

Comment:
Common types, interfaces and consts not specific to Lambda are moved to `types.ts`